client/src/RatingsAndReviews/RatingsBreakdown/BreakdownClickable.jsx

```diff
@@ -17,7 +17,8 @@ const BreakdownClickable = (props) => {
 BreakdownClickable.propTypes = {
   stars: PropTypes.number,
   percent: PropTypes.number,
-  count: PropTypes.number
+  count: PropTypes.number,
+  starFilterClick: PropTypes.func
```

Comment:
I notice that you have propTypes camelCased on line 17 but capitalized both words just below. I've been using syntax like 
`BreakdownClickable.propTypes = {
  stars: propTypes.number,
  percent: propTypes.number
}
`.
Not sure if it prop-types works the same either way, but if it does, I'll adjust mine to match your syntax

Reply:
yeah that is just the way I saw it in the docs. 

.propTypes for adding propTypes to a class / functional component 
vs 
PropTypes for accessing the react PropTypes class

Reply:
ahh gotcha, thanks for clarification!